examples/transaction-statement/transaction_statement.description

```diff
@@ -1,5 +1,5 @@
 // Ballerina provides support for interacting with a transaction manager.
-// Compile-time guarantees that transactions are bracketed with begin and commit or rollback.
-// The region in the middle, is typed as being a transactional context.
-// Ballerina does not have a transactional memory and includes transaction manager.
-// Current transaction is a part of the execution context of a strand.
+// Compile-time guarantees that transactions are bracketed with `begin` and `commit` or `rollback`.
```

Comment:
Not sure about `begin` @pcnfernando @gimantha can you verify?

Reply:
There's no begin keyword, right? We shouldn't be emphasizing begin.